home/screens/UserSettingsScreen.tsx

```diff
@@ -104,6 +109,41 @@ function MenuGestureItem() {
   );
 }
 
+function TrackingItem() {
+  const [showTrackingItem, setShowTrackingItem] = React.useState(false);
+  React.useEffect(() => {
+    (async () => {
+      const { status } = await Tracking.getTrackingPermissionsAsync();
+      setShowTrackingItem(status === 'undetermined');
+    })();
+  }, [showTrackingItem]);
+
+  return showTrackingItem ? (
+    <View style={styles.marginTop}>
+      <SectionHeader title="Tracking" />
+      <ListItem
+        title="Allow projects to track device data"
```

Comment:
add `last` prop here like on `SignOutItem` so the row has the correct bottom border